google-cloud-spanner-hibernate-dialect/src/test/java/com/google/cloud/spanner/hibernate/GeneratedCreateTableStatementsTests.java

```diff
@@ -245,4 +248,19 @@ public void testCreateUniqueIndexes_oneToMany() throws SQLException {
         "RUN BATCH"
     );
   }
+
+  @Test
+  public void testRunBatchFails() {
```

Comment:
Could you also write a test for `START BATCH DDL`, or make this one a parameterized test.

Reply:
Sure.